app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt

```diff
@@ -356,7 +357,7 @@ class SuggestedEditsImageTagsFragment : SuggestedEditsItemFragment(), CompoundBu
                 .subscribe({ token ->
                     val mId = "M" + page!!.pageId
                     var claimStr = "{\"claims\":["
-                    var commentStr = "/* add-depicts: "
+                    var commentStr = "/* add-depicts: " + DescriptionEditFragment.SUGGESTED_EDITS_IMAGE_TAGS_COMMENT
```

Comment:
Here we are building a list into `commentStr` of the image tags that were added. We should not be putting our additional #comment in the middle of the list. It should come at the end, after the `commentStr` is closed with `*/`